packages/ui/src/components/Checkbox/style.less

```diff
@@ -125,38 +69,49 @@
     }
   }
 
-  &:hover > .condo-checkbox > .condo-checkbox-inner {
-    border-color: @condo-global-color-gray-7;
+  &-disabled .condo-checkbox-inner {
+    opacity: 0.5;
   }
 
-  &:hover > .condo-checkbox-checked > .condo-checkbox-inner::before {
-    border-radius: @checkbox-inner-border-radius;
-  }
+  .condo-checkbox.condo-checkbox-indeterminate > .condo-checkbox-inner {
+    background: @condo-global-color-brand-gradient-1;
 
-  .condo-checkbox-indeterminate > .condo-checkbox-input:focus + .condo-checkbox-inner::before {
-    .condo-checkbox-inner-layer(@condo-global-color-brand-gradient-1, @checkbox-inner-border-radius);
+    &::after {
+      width: 14px;
```

Comment:
why this size? Is it related to font? or why `x % 4 !== 0`

Reply:
Will be 12px, does not related on typography